### ISSUE TITLE
how to delete

### ISSUE THREAD

**Issue by n6ac** [fastlane/fastlane#18466]
Mac on Mojave here, Terminal starts with "nexport: command not found"
Somewhere found clue that maybe it is due to an old version of fast lane
I can not find anywhere how to delete fast lane which I do not use.
Please add how to delete to the docs?

**Comment by minuscorp** [comment]
If you installed through brew: `brew uninstall fastlane`
If you installed through Rubygems:
`gem uninstall fastlane` (sudo might be needed).

**Comment by n6ac** [comment]
Thank you!

Apparently not installed, and thus not the source of then export issue …

,,, Alfred

Last login: Sun Mar 28 19:01:22 on ttys000
-bash: nexport: command not found
Mini4:~ myname$ brew uninstall fastlane
Error: No available formula or cask with the name "fastlane".

**Comment by minuscorp** [comment]
I'll close it then, as it apparently has no relation with fastlane, thank you!

**Comment by n6ac** [comment]
Jorge –

Thank you.

Incidentally, fastlane was a suspect because it was involved in one of the 2 responses to a Google search for
	macos terminal "-bash: nexport: command not found”
Apparently something I installed that was supposed to add these lines to .bash_profile –
	export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk-15.0.1.jdk/Contents/Home
	export PATH="/usr/local/opt/ruby/bin:$PATH"
– started with “\n” for new line, but was garbled leaving 
	\nexport JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk-15.0.1.jdk/Contents/Homeexport PATH="/usr/local/opt/ruby/bin:$PATH"

I am not sure what it was, but it led me to lessons about “.bash_profile” so in the end I guess that is a good thing.
And I learned that there is NO “nexport" command!

Sorry to bother you!

… Alfred
